E2E-tests/secrets/substrate/demo/keys/dave/partner_chain.skey.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Which keys are these? What is demo? I thought we renamed it to local env. So this might be eligible for deletion.